pyface/tests/test_about_dialog.py

```diff
@@ -26,17 +26,20 @@ def tearDown(self):
         if self.dialog.control is not None:
             with self.delete_widget(self.dialog.control):
                 self.dialog.destroy()
+        self.dialog = None
         GuiTestAssistant.tearDown(self)
 
     def test_create(self):
         # test that creation and destruction works as expected
         self.dialog._create()
         self.gui.process_events()
         self.dialog.destroy()
+        self.gui.process_events()
```

Comment:
if a gui_test_assistant exists the correct thing is to use `self.event_loop()` or `with self.event_loop()`